api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/alerts_controller.rb

```diff
@@ -12,20 +12,23 @@ module MnoEnterprise::Concerns::Controllers::Jpi::V1::Impac::AlertsController
 
   # GET /jpi/v1/impac/alerts
   def index
-    @alerts = current_user.alerts
+    u = current_user.load_required(:alerts)
+    @alerts = u.alerts
   end
 
   # POST /jpi/v1/impac/kpis/:kpi_id/alerts
   def create
     return render_bad_request('attach alert to kpi', 'no alert specified') unless params.require(:alert)
-    return render_not_found('kpi') unless kpi_alert.kpi
+    return render_not_found('kpi') unless  Mno::Kpi.find_one(params.require(:kpi_id))
 
-    authorize! :manage_alert, kpi_alert
+    # TODO: Manage authorization
+    #authorize! :manage_alert, kpi_alert
```

Comment:
Yeah, I need to record that one. It's a bit difficult as the authorization was relying on the chain call from the widget to retrieve the user.